### PR TITLE
Jl/backport 306 v1.14

### PR DESCRIPTION
**Pull request by jedleggett**
#### Summary of Changes
Backport https://github.com/jito-foundation/jito-solana/pull/306

### CODE REVIEW

**Review by buffalu — APPROVED** [review]

File: validator/src/main.rs [hunk 1]
```diff
@@ -1946,7 +1946,7 @@ pub fn main() {
                 .long("shred-receiver-address")
                 .value_name("SHRED_RECEIVER_ADDRESS")
                 .takes_value(true)
-                .help("Shred receiver listening address")
+                .help("Validator will forward all shreds to this address in addition to normal turbine operation. Set to empty string to disable.")
```
Comment (segfaultdoc) [inline validator/src/main.rs:1949]: nit: omit to disable?

File: validator/src/main.rs [hunk 2]
```diff
@@ -2169,6 +2169,18 @@ pub fn main() {
...
+        .subcommand(
+            SubCommand::with_name("set-shred-receiver-address")
+                .about("Changes shred receiver address")
+                .arg(
+                    Arg::with_name("shred_receiver_address")
+                        .long("shred-receiver-address")
+                        .value_name("SHRED_RECEIVER_ADDRESS")
+                        .takes_value(true)
+                        .help("Validator will forward all shreds to this address in addition to normal turbine operation. Set to empty string to disable.")
+                        .required(true)
```
Comment (segfaultdoc) [inline validator/src/main.rs:2181]: must it be required?

**Review by segfaultdoc — APPROVED** [review]

File: validator/src/main.rs (hunk 2 above)
Comment (jedleggett) [reply validator/src/main.rs:2181]: i think in the subcommand, we want it required,

see discussion here
https://github.com/jito-foundation/jito-solana/pull/304#discussion_r1197895880

File: validator/src/main.rs (hunk 1 above)
Comment (jedleggett) [reply validator/src/main.rs:1949]: yeah, if they omit at startup it will be disabled, will fix help

**Pull request merged**